### DISCUSSION TITLE
Changing code snippet timestamps embedded in image for otherwise static content

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#1160] [General]
https://github.com/jayvdb/jayvdb/commit/6856c7c5f71512770b07620bf6ba6103c6d3040e is showing a change when only the timestamp embedded in the image was modified.

In any case, I am not sure if that was the timestamp of the action run, or of the commit it is describing.  

If the later, why did the commit with the same hash change timestamp.

If the former, I vaguely recall there was a discussion or issue around 'dont update an image if only the metadata changed', which was likely about `base`, not `code` and other plugins.  Perhaps a simpler solution is an option to disable the timestamp.

**Comment by lowlighter** [comment]
It's using the commit it is describing:
https://github.com/lowlighter/metrics/blob/6699d7ecc26d8a60170a5ae74bb2367518340efe/source/plugins/code/index.mjs#L41

Not sure why the same hash had a timestamp change though 😕
